otelcolbuilder/.otelcol-builder.yaml

```diff
@@ -176,3 +176,14 @@ replaces:
   # version which gets then translated into a replace in go.mod file.
   # This does not replace the version that sumologicexporter depends on.
   - github.com/SumoLogic/sumologic-otel-collector/pkg/extension/sumologicextension => ../../pkg/extension/sumologicextension
+
+  # This fixes panic for changing fingerprint_size with already calculated fingerprint
+  # TODO: remove this when the following change is released upstream:
+  #
+  # - fix(pkg/stanza/input/file/reader): skip building fingerprint in case of configuration change
+  #   https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/10485
+  #
+  # For now, cherry-picked changes are being used:
+  # commit: https://github.com/SumoLogic/opentelemetry-collector-contrib/commit/6528176f3fd636ac770c5ebf1417718e84a7280e
+  # branch: https://github.com/SumoLogic/opentelemetry-collector-contrib/tree/v0.52.0-stanza
+  - github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza => github.com/SumoLogic/opentelemetry-collector-contrib/pkg/stanza 6528176f3fd636ac770c5ebf1417718e84a7280e
```

Comment:
Another fork to maintain 😞 Hopefully not for long?

Reply:
Only until the fix makes its way upstream.

Reply:
I will update `release.md`, just in case :sweat_smile:

Reply:
updated as `easy-to-drop` change, so not ideal, but easy to drop :D